### DISCUSSION TITLE
Multiple embeds and `embeds` array

### DISCUSSION THREAD

**Discussion by Shinsekai7** [Glitchii/embedbuilder#19] [Ideas]
Just like fields, it would be great to have the ability to add multiple embeds.

**Comment by Glitchii** [comment]
Added to the [board](https://github.com/Glitchii/embedbuilder/projects/3)

**Comment by Glitchii** [comment]
Multi-embeds added on commit 270e8a94328842f308b34a6f54a3cddb4a6f919c. Enable the feature using the `multiembeds` parameter, or the `multiEmbeds` option, or set the `multiEmbeds` item in local storage, or through the menu.

![image](https://user-images.githubusercontent.com/46096865/189742143-3176091f-fda5-4686-840f-62a30d8cef08.png)
